src/app/home/home.component.ts

```diff
@@ -15,14 +17,18 @@ import * as authActions from '../reducers/auth/redux/auth.actions';
 })
 export class HomeComponent extends WithUnsubscribe() implements OnInit {
   public disableSecurityGroups = false;
-  public isSidenavVisible$ = this.store.pipe(select(layoutSelectors.isSidenavVisible));
-  public allowReorderingSidenav$ = this.store.pipe(select(configSelectors.get('allowReorderingSidenav')));
+  public routes$: Observable<Route[]> = this.navRoutesService.getRoutes();
+  public currentRoute$: Observable<Route> = this.navRoutesService.getCurrentRoute();
+  public subroutes$: Observable<Subroute[]> = this.navRoutesService.getSubroutes();
```

Comment:
I'm not quite sure that it should be initiated that way. You can find `this.navRoutesService` only in the constructor, and it's not so obvious.

Also, this initialization relies on code generation mechanisms which can be changed later by the developers of TypeScript. I suggest moving this logic into the constructor.